src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -578,13 +597,23 @@ namespace NServiceBus
         [System.ObsoleteAttribute("Please use `StartAsync()` instead. Will be removed in version 7.0.0.", true)]
         public static NServiceBus.IBus Start(this NServiceBus.IStartableBus bus) { }
     }
+    public interface IStartableEndpoint
+    {
+        System.Threading.Tasks.Task<NServiceBus.IStoppableEndpoint> StartAsync();
+    }
+    public interface IStoppableEndpoint
```

Comment:
it represents the main interface users are working with after starting the endpoint. Always using a `IStoppableEndpoint` (e.g. using DI) makes it weird I think.